From: Mike Turquette <mturquette@ti.com>
Date: Tue, 24 May 2011 17:10:32 +0000 (-0500)
Subject: OMAP: PM: mute warn from min_bus_tput constraint
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=19249d6a150614aac3d9f2ee981f446ec0c3a3b9

OMAP: PM: mute warn from min_bus_tput constraint

When the omap_device_set_rate function is called from
omap_pm_set_min_bus_tput and fails, a warn is printed.  This behavior is
normal and expected in DPLL cascading mode, so the warn has been converted
to pr_debug to minimize console noise.

Change-Id: Ic259bf4997890a8848276effe383497a7398f049
Signed-off-by: Mike Turquette <mturquette@ti.com>
---

Index: kernel/arch/arm/plat-omap/omap-pm.c
===================================================================
--- kernel.orig/arch/arm/plat-omap/omap-pm.c	2011-05-27 23:13:49.527266748 +0900
+++ kernel/arch/arm/plat-omap/omap-pm.c	2011-05-27 23:28:47.659598199 +0900
@@ -312,7 +312,7 @@
 	ret = omap_device_set_rate(&dummy_l3_dev, l3_dev, target_level);
 
 	if (ret)
-		pr_err("Unable to change level for interconnect bandwidth to %ld\n",
+		pr_debug("Unable to change level for interconnect bandwidth to %ld\n",
 			target_level);
 unlock:
 	mutex_unlock(&bus_tput_mutex);
